lib/pbench/agent/tool_meister.py

```diff
@@ -305,12 +303,9 @@ def start(self, tool_dir: Path):
             self.stop_process is None
         ), f"Tool({self.name}) has an unexpected stop process running"
 
-        args = [
-            self.tool_script,
-            "--start",
-            f"--dir={tool_dir}",
-            self.tool_opts,
-        ]
+        args = [self.tool_script, "--start", f"--dir={tool_dir}"] + shlex.split(
+            self.tool_opts
+        )
```

Comment:
Instead of rearranging the code to make `Black` not mangle it, can we just dump `Black` instead? :smirk:

In any case, will do (while holding my nose).